corehq/apps/users/bulk_download.py

```diff
@@ -43,84 +47,125 @@ def build_data_headers(keys, header_prefix='data'):
     )
 
 
-def parse_users(group_memoizer, domain, user_filters, task=None, total_count=None):
+def get_devices(user):
+    """
+    Returns a comma-separated list of IMEI numbers of the user's devices, sorted with most-recently-used first
+    """
+    return ', '.join([device.device_id for device in sorted(
+        user.devices, key=lambda d: d.last_used, reverse=True
+    )])
+
+
+def make_mobile_user_dict(user, group_names, location_cache, domain, fields_definition):
+    model_data = {}
```

Comment:
what are these two lines for? They seem to be immediately set to something else.